src/Microsoft.AspNet.WebUtilities/Encoders/UnicodeHelpers.cs

```diff
@@ -0,0 +1,164 @@
+﻿// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Diagnostics;
+using System.Reflection;
+using System.Runtime.CompilerServices;
+using System.Threading;
+
+namespace Microsoft.AspNet.WebUtilities.Encoders
```

Comment:
nits:
- a few long lines in this class can be wrapped
- needs a bit more `var`.  same for last couple of methods in `UnicodeFilters`